flutter_modular/lib/src/presenters/navigation/modular_router_delegate.dart

```diff
@@ -121,7 +127,8 @@ class ModularRouterDelegate extends RouterDelegate<ModularRoute>
     final trash = <String>[];
 
     injectMap.forEach((key, module) {
-      module.paths.remove(path);
+      // Removes all duplicate routes
+      module.paths.removeWhere((element) => element == path);
```

Comment:
Here. We will have problems in Multiples pages, for example. 2x PushNamed() wtth same route path.

Reply:
Yeah... this passed in my mind, but I thought that this situation will not be possible... I'll check for another solution. I did that because modular was adding two routes after one navigation in some situations.

Reply:
Hum Cool!
Thanks for all support!

Reply:
Any progress about this?

Reply:
Not yet, I was a bit busy, but I'll work on this in this weekend

Reply:
Now, I believe that these changes will work well!